src/NHibernate.Test/NHibernate.Test.csproj

```diff
@@ -74,12 +74,13 @@
     <Reference Include="System.Threading.Tasks" />
     <Reference Include="System.Data.OracleClient" />
     <PackageReference Include="Microsoft.SqlServer.Compact" Version="4.0.8876.1" />
-    <PackageReference Include="Oracle.ManagedDataAccess" Version="12.1.2400" />
+    <PackageReference Include="Oracle.ManagedDataAccess" Version="19.11.0" />
   </ItemGroup>
   <ItemGroup Condition="'$(TargetFramework)'=='netcoreapp2.0'">
     <PackageReference Include="System.CodeDom" Version="4.7.0" />
     <PackageReference Include="System.Data.SqlClient" Version="4.8.2" />
     <PackageReference Include="System.Data.OracleClient" Version="1.0.8" />
+    <PackageReference Include="Oracle.ManagedDataAccess.Core" Version="2.19.110" />
```

Comment:
I have not checked. But should `Oracle.ManagedDataAccess.Core` also work for .NET Framework ?

Reply:
Well I guess it should for 2.x as this version still supports .NET Standard 2.0. But they dropped it in latest 3.x Core versions. So this lib split would still be required if we want to keep up to date versions for both frameworks.

Reply:
Also IMO it's good to cover both libs in tests...

Reply:
Makes sense.